addons/spreadsheet/static/src/pivot/pivot_model.js

```diff
@@ -334,6 +334,15 @@ export class OdooPivotModel extends PivotModel {
         return displayName;
     }
 
+    _normalize(gb) {
```

Comment:
gb ? 

![image](https://github.com/user-attachments/assets/ef2ddf0c-b2ec-434d-8906-c6d0dbf1c001)

There _might_ exist better variable names floating somewhere in the aether

Reply:
groupBy, I used the same as the parent method. But yes I will rename it